index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<head>
+    <title>
+        freeCodeCamp Survey Form
+    </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">
+        Thank you for
+        taking the time to help us
+        improve the platform
+    </p>
+    <form id="survey-form">
+        <label for="name" id = "name-label">Name</label>
```

Comment:
It is always good not leaving blank spaces between a property and its value. Kindly try to change that on the `id` property. It might look like this:
`id="name-label"`
With this in mind, it looks now prettier and easier to read, isn't it? :smile: